### PR TITLE
Client sdk call reject rename

### PR DESCRIPTION
**Pull request by pardel**
## Description

iOS and Android Client SDK got call.decline renamed to call.reject. This PR propagates this to the docs.

## Deploy Notes

Nothing special for deployment.

### CODE REVIEW

File: _documentation/client-sdk/in-app-voice/building-blocks/reject-call.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Reject a Call
+navigation_weight: 5
+---
+
+# Reject a Call
+
+In this building block you will see how to reject a Call, before it was answered.
+For implementation in Android, see the [Hangup building block](hangup-call)
```
Comment (brittBarak) [inline _documentation/client-sdk/in-app-voice/building-blocks/reject-call.md]: @mheap @tbedford is this what you meant?

Comment (tbedford) [reply _documentation/client-sdk/in-app-voice/building-blocks/reject-call.md]: Yes, although that link is not quite right.

Comment (pardel) [reply _documentation/client-sdk/in-app-voice/building-blocks/reject-call.md]: Resolved by @mheap 
https://github.com/Nexmo/nexmo-developer/pull/1428/commits/9cda938315e97ec6d362a2f3e8200d1c2b4f7c02

**Comment by mheap** [comment]
I've updated the link on the reject block for Android so that it's a full path from the root URL (this is important for other NDP functionality). Merging as Britt's approved

**Pull request merged**